avl_tree/AvlTree.java

```diff
@@ -0,0 +1,306 @@
+import java.util.Scanner;
+import java.util.NoSuchElementException;
+import java.util.InputMismatchException;
+
+public class AvlTree {
+
+	public static void main(String[] args) {
+		AVL avl = new AVL();
+		Scanner inp = new Scanner(System.in);
+		while (true) {

```

Comment:
I read it, So Mr. @Monal5031 enlighten me with your divine knowledge, what you want me to do here.